### DISCUSSION TITLE
Setting up UERANSIM with ONF's Aether?

### DISCUSSION THREAD

**Discussion by ahass29** [aligungr/UERANSIM#567] [Q&A]
Hello. Has anyone tried to use the UERANSIM with ONF's Aether as the Core Network? I am lost and I don't really know where to start. Thanks!

**Comment by vlanx** [comment]
They've added a [documentation page](https://docs.sd-core.opennetworking.org/master/deployment/deploymentueransim.html) on the SD-Core project about it.

**Reply by Prakrati2000** [reply]
The page is not available.

**Reply by MarcusJoha** [reply]
https://docs.sd-core.aetherproject.org/main/deployment/deploymentueransim.html

**Comment by MarcusJoha** [comment]
Recommend deploying with the aether-onramp. Follow the guide for configuration. Run `make aether-k8s-install` and `make aether-5gc-install`. Check that it works: `make aether-pingall`. Configure your UERANSIM. Remember to add route `ip route add 192.168.252.3 via <IP of ueransim-server>`. Check internet connection: `ping -I uesimtun0 8.8.8.8 -c 3`.
